pom.xml

```diff
@@ -363,7 +364,7 @@
                   var artifactId = project.getProperty("project.artifactId");
                   var version = project.getProperty("project.version");
 
-                  var cfgFile = new java.io.File(targetDir, artifactId + "-" + version + ".json");
+                  var cfgFile = new java.io.File(targetDir, artifactId + ".json");
```

Comment:
Since the module has different version, the version will be appended the kafka version we use, so I remove the version here to generate the correct name for the jar and json